webpage/conda.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Installing the IOOS conda environment
+layout: single
+---
```

Comment:
@jbosch-noaa this example shows the beauty of Markdown, `Jekyll`, and a single theme. I only copied the wiki, which is already in Markdown and added the "front matter"\*, and it is done! We do not need to worry about about styling, HTML, CSS, nothing.

\* The header that Jekyll needs to understand what type of page is this.